research/H1/FPN.md

```diff
@@ -0,0 +1,106 @@
+# Label: H1, Preamble: "FPN"
+
+Flight Plan
+
+## Format
+
+The message begins with `FPN/` followed by a status. After that, it is a list of key/value pairs delimited by `:`. The final 4 characters are a checksum (algorithm unknown).
+
+### Known Statuses
+
+key | value
+---|-----
+RI | Route Inactive
+FNXXXX | Flight Number XXXX
+
+### Known Keys
+
+key | value
+---|-----
+A  | Arrival Procedure
+AA | Arrival Airport
+AP | Approach Procedure (?)
+CR | Current Route (?)
```

Comment:
in case it's not obvious - the `(?)` is to indicate i'm not 100% sure

Reply:
Total acceptable. If anyone else (or myself) identifies, we can update later.